<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>single_message_box</class>
 <widget class="QDialog" name="single_message_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextDump</string>
  </property>
  <widget class="QLabel" name="sender_timestam_lable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message from: PERSON At: TIME</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="message_text_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;I am a message&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy To Clip Board</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
